src/OrleansRuntime/Scheduler/WorkerPoolThread.cs

```diff
@@ -133,9 +137,9 @@ protected override void Run()
 #endif
                         // Get some work to do
                         IWorkItem todo;
-
-                        todo = IsSystem ? scheduler.RunQueue.GetSystem(Cts.Token, maxWorkQueueWait) : 
-                            scheduler.RunQueue.Get(Cts.Token, maxWorkQueueWait);
+                        todo = null;
+                        //todo = IsSystem ? scheduler.RunQueue.GetSystem(Cts.Token, maxWorkQueueWait) : 
```

Comment:
There is: https://github.com/dotnet/orleans/pull/2060/files#diff-f21e00f434dac5a42463e7cae5b772f7R49. WorkerPoolThread doesn't being used anymore.